payment/common/utils.go

```diff
@@ -48,3 +49,35 @@ func SaveTransactionsToDB(transactions []models.Transaction, collection string)
 
 	return nil
 }
+
+func UpdatePaymentProvider(policy *models.Policy, provider string) error {
+	networkNode := network.GetNetworkNodeByUid(policy.ProducerUid)
+	if networkNode == nil {
+		return fmt.Errorf("can't find network node for policy %s ", policy.Uid)
+	}
+	warrant := networkNode.GetWarrant()
+	if warrant == nil {
+		return fmt.Errorf("can't find warrant for network node %s ", networkNode.Uid)
+	}
+	flow := warrant.GetFlowName(policy.Name)
+	if flow == "" {
+		return fmt.Errorf("can't find a flow for policy %s", policy.Uid)
+	}
+
+	switch provider {
+	case models.ManualPaymentProvider:
+		if flow == models.RemittanceMgaFlow {
+			policy.Payment = provider
+			return nil
+		}
+		return fmt.Errorf("can't update payment because flow %s doesn't support provider %s", flow, provider)
+	case models.FabrickPaymentProvider:
+		if flow != models.RemittanceMgaFlow {
+			policy.Payment = provider
+			return nil
+		}
+		return fmt.Errorf("can't update payment because flow %s doesn't support provider %s", flow, provider)
+	default:
+		return fmt.Errorf("can't update payment because provider %s is not supported", provider)
+	}
+}
```

Comment:
As I've anticipated in our call, I would change this function according to these steps:

`1. check if policy channel is network
     a. if so, get node info (node+warrant)
     b. check if input provider is compatible with node flow
         - if not, error
  2. set policy provider to input provider        
`